.github/CODEOWNERS

```diff
@@ -129,6 +129,7 @@
 /pkg/metadata/                          @DataDog/agent-core
 /pkg/metrics/                           @DataDog/agent-core
 /pkg/serializer/                        @DataDog/agent-core
+/pkg/serverless/                        @DataDog/serverless
```

Comment:
Can you add this team here too https://github.com/DataDog/datadog-agent/blob/bc7f38037c7500790171fc9419eea74b52c77a3d/tasks/pipeline.py#L229 ?